interfaces/builtin/basedeclaration.go

```diff
@@ -190,6 +193,9 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  classic-dimension:
+    allow-installation: false
+    deny-auto-connection: true
```

Comment:
Ups, sorry :/ And thanks for spotting this. I really should not push code late at night. Fixed now.